Assets/Scripts/VoicelineManager.cs

```diff
@@ -86,16 +91,16 @@ protected void _Play(int voicelineIndex)
             {
                 bemisAudioSource.clip = voiceline;
                 bemisAudioSource.PlayDelayed(1f);
+                PlaybackHistory.AddRecord(voicelineIndex);
             }
-            PlaybackHistory.AddRecord(voicelineIndex);
         }
 
         public abstract void Play();
     }
 
     private class Voiceline : AVoiceline
     {
-        private int voicelineIndex;
+        public readonly int voicelineIndex;
```

Comment:
If we're switching to names (preferably enumerated as mentioned above), then all the index references should be replaced with `.name`

Reply:
let's just keep them as indices for now, then fix up the voiceline system come endless mode

Reply:
Then let's be consistent. The persistent voicelines are referenced by their clip name...